### DISCUSSION TITLE
On the Issue of Language Selection

### DISCUSSION THREAD

**Discussion by huangkunh** [GDQuest/learn-gdscript#984] [Q&A] [answered]
I wish to incorporate a Chinese language translation into this project. I noticed that the i18n folder includes zh_Hans files. Yet, upon opening the downloaded project, the Chinese language option is absent. The version I downloaded is "Learn GDScript 1.5.2". Can you explain why this occurs?
![image](https://github.com/user-attachments/assets/a9e28877-ced4-4a1b-b46b-6cf821f18c76)

**Comment by NathanLovato** [comment, marked as answer]
Yes, there are no Chinese fonts in the app, so Chinese characters cannot be displayed. It'd need a developer to add support for this. You can find more details in this issue: https://github.com/GDQuest/learn-gdscript/issues/497

And in this pull request: https://github.com/GDQuest/learn-gdscript/pull/688

**Reply by huangkunh** [reply]
I am delighted to receive your response. I now understand the situation.
